pkgs/applications/graphics/krita/wrapper.nix

```diff
@@ -0,0 +1,26 @@
+{ callPackage
+, symlinkJoin
+, makeWrapper
+, krita-unwrapped
+, krita-plugin-gmic
+, plugins ? [
```

Comment:
Not sure about the default here

Reply:
The plugin comes by default in the flatpak and appimage versions, so I went with that. 

I realize it's easier to add plugins outside of those packaging formats, but it didn't feel right to leave users with a partially functioning program by default. 

Also, gmic used to be included by default in nixpkgs' krita before v5 IIRC.

Reply:
That sounds fine to me but probably a very short comment doesn't hurt